marshal/consumer.go

```diff
@@ -622,28 +630,37 @@ func (c *Consumer) Terminate(release bool) bool {
 // consumer. It should be relevent only when ClaimEntireTopic is set
 func (c *Consumer) GetCurrentTopicClaims() (map[string]bool, error) {
 	if !c.options.ClaimEntireTopic {
-		err := fmt.Errorf("GetCurrentTopicClaims is only relevent when ClaimEntireTopic is set")
-		log.Error(err.Error())
-		return nil, err
+		return nil, errors.New(
+			"GetCurrentTopicClaims requires options.ClaimEntireTopic be set")
+	}
+
+	claimedTopics := make(map[string]bool)
+	if c.Terminated() {
```

Comment:
We can terminate w/o release, and in that case we still have claims. I am not sure it matters either way but it seemed cleaner if a terminated consumer says it owns nothing. I can revert?